### PR TITLE
Order API

### PR DESCRIPTION
**Pull request by Vercjames**
testing new branch

### CODE REVIEW

**Comment by al1271** [comment]
I approve, don't see anything wrong.

File: routes/api/orderAPI.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+// Gravity Application Server | orderAPI
+// =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+module.exports = function (express) {
+  var router = express.Router();
```
Comment (icarlosmendez) [inline routes/api/orderAPI.js]: Per the airbnb style guide, all instances of 'var' should be changed to 'const'.

File: routes/api/orderAPI.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    // http://localhost:3000/orderAPI/details
+    var orderDetails = {};
+    var serverMessage = '';
+    var clientOrder = req.body;
+
+    // Triggered server response when order is placed
+    console.log('Request For Order ' + clientOrder.orderId);
+
+
+    for(i = 0; i <= orderDatabase.length; i++){
```
Comment (icarlosmendez) [inline routes/api/orderAPI.js]: Whitespace: 18.2 Place 1 space before the leading brace.
Above this line you adhered to the standard but from here down you changed. Please update accordingly.

File: routes/api/orderAPI.js (hunk 1 above)
Comment (Vercjames) [inline routes/api/orderAPI.js]: that is in es6, not es5. 'const' notation won't work until they recieve browser support hopefully in 2016. make sure your on the es5 airbnb style guide.

https://github.com/airbnb/javascript/tree/master/es5
not
https://github.com/airbnb/javascript

File: routes/api/orderAPI.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+    {orderId: 10001, status: 'warehouse', stock : [
+      {item : 'usbDrive', price : 12.00, quality : 2, location : {robotId : 1424, currentStock : '98', restock : '50', status: 'delivered'}},
+      {item : 'Computer', price : 90.00, quality : 1, location : {robotId : 1777, currentStock : '44', restock : '20', status: 'delivered'}},
+      {item : 'Keyboard', price : 25.00, quality : 1, location : {robotId : 1982, currentStock : '17', restock : '15', status: 'inTransit'}}
+    ], recipient : {name : 'Brandy Bergh', address : '3300 University Blvd, Winter Park, FL 32792', contact : 555-555-5555}}
+  ];
+
+
+
```
Comment (icarlosmendez) [inline routes/api/orderAPI.js]: Personal recommendation. No more than two blank lines between major code blocks. Guide says definitely one but I like two between components or major portions.

File: routes/api/orderAPI.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+      serverMessage : serverMessage,
+      Details: orderDetails
+    });
+  });
+
+// =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+router.post('/placement', function(req, res) {
+  // TODO: accept order being placed externally
+});
+
```
Comment (icarlosmendez) [inline routes/api/orderAPI.js]: Please bring this down to no more than two blank lines.

File: routes/api/orderAPI.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+
+
+
+
```
Comment (icarlosmendez) [inline routes/api/orderAPI.js]: Please bring this down to no more than two blank lines.

File: server/routes/orderAPI.js [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+  var orderPlacement = req.body;
+  orderTracking = { status : 'packing', timestamp : 'today'};
+  orderRecipient = orderPlacement.order.recipient;
+  orderUnits = orderPlacement.order.units;
+
+
+  var placement = {};
+  placement.tracking = orderTracking;
+  placement.recipient = orderRecipient;
+  placement.unit = orderUnits;
```
Comment (al1271) [inline server/routes/orderAPI.js]: Placement isn't a very good definition for what is being done here, maybe try thinking of another variable name.

**Comment by reactivepixel** [comment]
Merged with Love by Alex and Chris

**Pull request merged**